core/pom.xml

```diff
@@ -292,6 +292,16 @@
     <dependency>
       <groupId>io.dropwizard.metrics</groupId>
       <artifactId>metrics-graphite</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>com.rabbitmq</groupId>
+          <artifactId>amqp-client</artifactId>
```

Comment:
This would otherwise be a new dependency on RabbitMQ, but, my hunch is that it is not necessary for Spark.